WordPress/gutenberg
# Add page attributes menu order to sidebar

Partially addresses #991 

This pull request seeks to add a Page Attributes panel to the sidebar for post types supporting the `page-attributes` support type.

![Page Attributes](https://user-images.githubusercontent.com/1779930/28420250-89ebc30a-6d2f-11e7-838a-de1861f5ca9d.png)

__Testing instructions:__

Verify that a Page Attributes panel is shown for post types supporting `page-attributes`, and not shown for other post types. Ensure that only valid input (numbers) is accepted, and that the value is persisted when saving the post.

1. Navigate to Pages > All Pages
2. Hover an existing Page and click Gutenberg
3. In sidebar, note Page Attributes panel
4. Expand Page Attributes and enter numeric order
5. Press Save
6. Refresh page
7. Note order persisted
8. Repeat steps 1 - 3 for Posts, noting that Page Attributes is _not_ present

Ensure that unit tests pass:

```
npm run test-unit editor/sidebar/page-attributes/test/index.js
```
